src/components/SecretsPanel.tsx

```diff
@@ -0,0 +1,146 @@
+import React, { useState, useEffect } from 'react';
+import { ReactWidget } from '@jupyterlab/apputils';
+
+import '../../style/base.css';
+import { ISecret, ISecretsManager } from '../token';
+
+interface ISecretsPanelProps {
+  manager: ISecretsManager;
+}
+
+const EyeIcon = () => (
```

Comment:
I didn't find a good import for these eye icons, please let me know if any better eye icons